bigtable-hbase-1.x-parent/bigtable-hbase-1.x/pom.xml

```diff
@@ -137,6 +137,14 @@ limitations under the License.
       <scope>compile</scope>
     </dependency>
 
+    <!--  TODO: remove this dependency when upgraded through transitive dependency (http-client -> org.apache.httpcomponents:httpclient)
```

Comment:
Why is this coming up now? Should this be part of a different PR?

Reply:
I think this needs to be rebased

Reply:
Nevermind: I think the classes that this module depended on were all moved from core into the metrics module

Reply:
Took a closer look - this was added to fix the deps build: https://github.com/googleapis/java-bigtable-hbase/pull/2631/commits/bf9a9222fb99bb7d342cdc5e7d045062fabf3ba4 when switching the dependency from bigtable-client-core to metrics

Reply:
please rebase to minimize the noise

Reply:
Sorry, what I mean was this change is necessary since we moved code from core to metrics modules in this change.

Reply:
please rebase, this souldnt be necessary anymore